tests/thingpedia-integration/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM localhost/almond-cloud
+
+USER root
+
+COPY --chown=almond-cloud:almond-cloud ./tests /opt/almond-cloud/tests
+
+RUN curl -sL https://rpmfind.net/linux/epel/7/x86_64/Packages/l/libpmem-1.1-1.el7.x86_64.rpm  -o libpmem-1.1-1.el7.x86_64.rpm && \
+    curl -sS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | bash && \
+    dnf -y install libpmem-1.1-1.el7.x86_64.rpm  MariaDB-client && \
+    rm -f libpmem-1.1-1.el7.x86_64.rpm && rm -rf /var/cache/dnf
```

Comment:
Why do we need to install mariadb from third party repos? The base image OS (RHEL 8 UBI) should have mariadb in the normal repos.

Reply:
Please let me know how to get it. I couldn't find it. I also did a dnf search mariadb .

Reply:
Oh. I just looked too, mariadb is not part of the UBI images. This is fine then.